### PR TITLE
Call showAll on opening projects

### LINKED ISSUE #9902: New project not centered
When I create a new project the starting component should be centered in the window and zoomed 100%.

This is what it does today.
image
Sometimes the starting component can be off screen

### PR DESCRIPTION
**Pull request by vitvakatu**
### Pull Request Description

Fixes #9902 

https://github.com/enso-org/enso/assets/6566674/2eb2b0dd-610a-48e7-aeb6-6c9c423746db

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

File: app/gui2/src/components/GraphEditor.vue [hunk 1]
```diff
@@ -68,14 +68,31 @@ const suggestionDb = useSuggestionDbStore()
...
+    if (methodPtr != null) encodeMethodPointer(enc, methodPtr)
+  },
+  waitInitializationAndPanToAll,
+)
+
+const stopInitialization = ref<() => void>()
+function waitInitializationAndPanToAll() {
+  stopInitialization.value?.()
+  stopInitialization.value = watchEffect(() => {
+    const nodesCount = [...graphStore.db.nodeIdToNode.keys()].length
```
Comment (kazcw) [inline app/gui2/src/components/GraphEditor.vue]: `nodeIdToNode.size`?

File: app/gui2/src/components/GraphEditor.vue [hunk 2]
```diff
@@ -68,14 +68,31 @@ const suggestionDb = useSuggestionDbStore()
...
+    // - each project
+    // - each function within the project
+    encoding.writeVarString(enc, projectStore.name)
+    const methodPtr = graphStore.currentMethodPointer()
+    if (methodPtr != null) encodeMethodPointer(enc, methodPtr)
+  },
+  waitInitializationAndPanToAll,
+)
+
+const stopInitialization = ref<() => void>()
```
Comment (kazcw) [inline app/gui2/src/components/GraphEditor.vue]: Does this need to be reactive?

File: app/gui2/src/components/GraphEditor.vue [hunk 3]
```diff
@@ -68,14 +68,31 @@ const suggestionDb = useSuggestionDbStore()
...
+function waitInitializationAndPanToAll() {
+  stopInitialization.value?.()
+  stopInitialization.value = watchEffect(() => {
+    const nodesCount = [...graphStore.db.nodeIdToNode.keys()].length
+    const visibleNodeAreas = graphStore.visibleNodeAreas
+    if (nodesCount > 0 && visibleNodeAreas.length == nodesCount) {
+      zoomToSelected(true)
+      stopInitialization.value?.()
```
Comment (kazcw) [inline app/gui2/src/components/GraphEditor.vue]: Setting it to `undefined` would allow it to be freed

**Review by kazcw — APPROVED** [review]

File: app/gui2/src/components/GraphEditor.vue (hunk 1 above)
Comment (vitvakatu) [reply app/gui2/src/components/GraphEditor.vue]: Right, don’t ask how it appeared

**Comment by vitvakatu** [comment]
Sorry for ping guys, wrong rebase

**Pull request merged**